docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     ports:
       - 4444:4444
     volumes:
-      - $HOME/.msf4:/root/.msf4
+      - /tmp/.msf4:/home/msf/.msf4
```

Comment:
If we use `$HOME` and user is `root` mounting to docker sadly won't work and we'll get `EACCESS` error on msfconsole start even if we chmod 777 it, because parent dir is not accessible. So we have to make a decision where the share folder will be. Help wanted here.